cypress/support/step-definitions/content-steps.js

```diff
@@ -1,9 +1,7 @@
 import { contentBody, contentTitle, contentPreview } from '../../locators/content';
 
 const CONTENT_AS_LABEL = 'carbon-content--';
-const CONTENT_ALIGN_LABEL = `${CONTENT_AS_LABEL}align-`;
 const BODY_FULL_WIDTH_PARAMETR = `${CONTENT_AS_LABEL}body-full-width`;
```

Comment:
do we still need this stuff ?

Reply:
yes, but I've changed it. Instead of `const` I put it in step directly